test/blackbox-tests/test-cases/doc-browser.t/open

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+echo $@
```

Comment:
Adding something like
```
echo "Web browser received arg:"
```
might make the test easier to read. Same for the other one.

Better yet have a single copy and create symlinks so we don't have to duplicate.